docs/guides/08-interactivity-events.md

```diff
@@ -0,0 +1,411 @@
+    const latitude = center.lat.toFixed(6);
+    const bearing = map.getBearing().toFixed(0);
+    const zoom = map.getZoom().toFixed(2);
+    console.log(`Center: [${longitude}, ${latitude}] - Zoom: ${zoom} - Bearing: ${bearing}º`);
+};
+map.on('move', displayCenter);
+```
+> Check the console after loading your file in the browser, to see the first message: 'Map has loaded'. Then, interact with the map control and see the updated values for center & zoom. You can check Mapbox [Map reference](https://www.mapbox.com/mapbox-gl-js/api/#map) for more information on map events.
+
+The result should look like this:
+<div class="example-map">
+    <iframe
+        id="guides-interactivity-step-1"
+        src="/developers/carto-vl/examples/maps/guides/interactivity/step-1.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+
+### Layer events
+Once you have your basemap, you can start to work with your layers. And all [carto.Layer](/developers/carto-vl/reference/#cartolayer) objects have two events to listen to: `loaded` and `updated`.
+
+The use of the `loaded` event is pretty common, due to in most cases you need to load the data from an external server and that can take some time.
+
+Add this to your code to create a layer as usual, and add it to your map:
+```js
+carto.setDefaultAuth({ user: 'cartovl', apiKey: 'default_public' });
+const source = new carto.source.Dataset('populated_places');
+const viz = new carto.Viz();
+const layer = new carto.Layer('Cities', source, viz);
+layer.addTo(map);
+```
+
+Now add a `loaded` listener to the previous layer:
+```js
+layer.on('loaded', () => {
+    console.log('Cities layer has been loaded!');
+});
+```
+> This event could be useful for example to display some kind of loading animation over your map and then hiding it after the layer has loaded. Notice how the name of the event is `loaded`, not `load`.
+
+If you were using several layers, you could also have a single function to handle them all. For this case, **on** & **off** are available at `carto` namespace, expecting a list of layers like this: `carto.on('loaded', [layer1, layer2], () => { console.log('All layers have loaded'); })`.
+
+Regarding to the `updated` event, it can be useful for some other cases, where the layer's viz changes, for example when you are building an animation. See the [Playing with animations guide](/developers/carto-vl/guides/playing-with-animations).
+
+If you check your work now, it should look like this:
+<div class="example-map">
+    <iframe
+        id="guides-interactivity-step-2"
+        src="/developers/carto-vl/examples/maps/guides/interactivity/step-2.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+
+### Using dynamic variables
+*Variables* are a way to store and reuse expressions, and that can definitively help you when adding interactions to your visualization, so let's practice a bit with them.
```

Comment:
Now that we've a glossary, it may be useful to add this concept there as well (just a suggestion)

Reply:
Good idea. I have just added `Variables` to that document